omnigan/trainer.py

```diff
@@ -766,11 +819,18 @@ def train(self):
             self.save()
 
             if (
-                self.logger.epoch == self.opts.gen.p.pl4m_epoch
+                self.logger.epoch == self.opts.gen.p.pl4m_epoch - 1
                 and get_num_params(self.G.painter) > 0
             ):
                 self.use_pl4m = True
 
+            if self.logger.epoch == self.opts.train.kitti.epochs - 1:
+                self.switch_data(to="base")
+                self.kitti_pretrain = False
+
+            if self.logger.epoch == self.opts.train.pseudo.epochs - 1:
```

Comment:
Unify pseudo-labels training interface -> pseudo-signal is disabled after `opts.train.pseudo.epochs` epochs